### DISCUSSION TITLE
de.siegmar.logbackgelf.GelfHttpAppender gives errors without message

### DISCUSSION THREAD

**Discussion by gbarsan** [osiegmar/logback-gelf#109] [Q&A]
We are using this settings:

`<appender name="GELFNEW" class="de.siegmar.logbackgelf.GelfHttpAppender">`
`        <uri>https://logs.example.com/gelf</uri>`
`        <encoder class="de.siegmar.logbackgelf.GelfEncoder">`
`            <staticField>application:${application}</staticField>`
`        </encoder>`
` </appender>`
`<appender name="ASYNC_GELF" class="ch.qos.logback.classic.AsyncAppender">`
`        <appender-ref ref="GELFNEW"/>`
`        <queueSize>500</queueSize>`
`        <neverBlock>true</neverBlock>`
`</appender>`
`<root level="DEBUG">`
`        <appender-ref ref="ASYNC_GELF"/>`
`</root>`

But it leads to this error on STDOUT:

`12:08:02,466 |-ERROR in de.siegmar.logbackgelf.GelfHttpAppender[GELFNEW] - Invalid response from https://logs.example.com/gelf. Code: 413; Message:`

Does anyone know how to handle this? How can we solve it?

Many thanks for any help!

**Comment by osiegmar** [comment]
HTTP error 413 resolves to "Content Too Large". Graylog (or the underlaying Elasticsearch) typically throws this error if the message is over ~32 KiB. Maybe that helps? What does the Graylog server log tell you?

**Comment by gbarsan** [comment]
We're trying to figure out how to find the according entries and see what Graylog "says". At least on NGINX (which we use as reverse proxy/for SSL termination) we don't see any errors. We will also try to implement a custom message filter and limit the size.
